src/index.js

```diff
@@ -1,38 +1,35 @@
 const routes = require("./routes");
-const initialise = require("./initialise");
 const repositories = require("./repositories");
 
 /**
  * @param {import('probot').Application} app - Probot's Application class.
  */
 module.exports = async app => {
-	// Ensure that we crash if there's any error loading our API routes.
-	try {
-		await routes(app);
-	} catch (err) {
-		app.log.fatal("Failed to load API routes", err);
-		process.exit(1);
-	}
-
-	// Ensure that we crash Probot if we are unable to initialise.
-	const installation = await initialise(app).catch(err => {
-		app.log.fatal("Failed to initialise", err);
-		process.exit(1);
-	});
+	await routes(app);
 
-	async function refresh(context) {
-		const repositoryCount = await repositories.refresh(installation);
-		app.log.info(
-			`Refreshed the repository store. Total: ${repositoryCount}. Action: ${context
-				.payload.action || "Unknown"}`
-		);
+	const refresh = async github => {
+		await repositories.refresh(github);
+		app.log.debug("Refreshed the repositoryStore. Length:", repositories.list().length);
 	}
 
 	/**
 	 * On appropriate events, refresh the whole list of Tako repositories.
-	 * @see https://developer.github.com/webhooks/#events for a list of all GitHub webhook events.
+	 * @see https://developer.github.com/v3/activity/events/types/#repositoryevent
+	 * @see https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent
+	 */
+	app.on([
+		"repository",
```

Comment:
We can limit the events to only those we care about.

We don't care about `publicized` or `privatized`, and `created` and `deleted` should also come with a `installation_repositories` which we already capture.

```suggestion
		"repository.archived",
		"repository.unarchived",
```